src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -501,7 +501,7 @@ describe("Editor Page Component", () => {
         });
 
         it("sets selected tag and locked tags when hot key is pressed", async () => {
```

Comment:
since we're not testing anything relating to locked tags here, we should update the test description